collector/config.go

```diff
@@ -60,15 +80,67 @@ func (c Config) Validate() error {
 	return nil
 }
 
+// decryptPrivateKey returns a RSA private key from the PrivateKeyPath and PrivateKeyPassword fields
+// of the config.
+// Assumes that the private key is encrypted in PKCS #8 syntax, as is recommended by Snowflake
+func (c Config) decryptPrivateKey() (*rsa.PrivateKey, error) {
+	var parsedPrivateKey *rsa.PrivateKey
+	pk, err := os.ReadFile(c.PrivateKeyPath)
+	if err != nil {
+		fmt.Printf("Error opening file: %s", err)
```

Comment:
I'll try the logger route, I think since there are multiple points of failure in this method, just having one error message across the board at the collector level is probably not enough.

Reply:
With my other comment you'll see that you don't need to log here as this error gets logged later after it bubbles up right?

Reply:
Yep, that's on me. I'm still working out all the different methods and mechanisms for errors in go.